routemaster/services/deliver.rb

```diff
@@ -36,14 +45,24 @@ def call
         error = nil
         start_at = Routemaster.now
         begin
-        # send data
+          _maybe_throttle!
+          @subscriber.attempting_delivery(start_at)
```

Comment:
`Throttle` feels inconsistent here: it's reading the `last_attempted_at`, but that is written here in `Deliver`. How about 

```diff
-  _maybe_throttle!
- @subscriber.attempting_delivery(start_at)
+ _throttle.check!(start_at)
+ # ...
+ def _throttle
+   @_throttle ||= Services::Throttle.new(subscriber: @subscriber)
```